src/coreComponents/linearAlgebra/CMakeLists.txt

```diff
@@ -50,9 +50,20 @@ set( linearAlgebra_sources
      DofManager.cpp )
 
 if( BUILD_OBJ_LIBS )
-  set( dependencyList common blas lapack RAJA managers)
+  set( dependencyList mesh blas lapack RAJA)
```

Comment:
Yeah so why does functions depend on `MKL` but `linearAlgebra` only depends on `blas` and `lapack`?

Reply:
errr...excellent query Mary.